### PR TITLE
Upgrade flask to v1.0.3

### PR DESCRIPTION
**Pull request by fungjj92**
This PR adds images with the newest major version of flask, 1.0.3. This is coming up as part of https://github.com/azavea/usace-levee-safety-api/issues/206, where flask is among the dependencies getting upgraded.

I followed the README and successfully built a container, locally.
Screen Shot 2019-06-21 at 2 51 19 PM

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@ env:
   - VERSION=0.12 PYTHON_VERSION=3.6 VARIANT=slim
   - VERSION=0.12 PYTHON_VERSION=2.7 VARIANT=alpine
   - VERSION=0.12 PYTHON_VERSION=3.6 VARIANT=alpine
+  - VERSION=1.0.3 PYTHON_VERSION=2.7 VARIANT=slim
```
Comment (rbreslow) [inline .travis.yml]: I would drop the patch version from the build matrix. This is inline with how we're handling the `0.x` series in this repo, and will allow applications utilizing the `1.0` series image to automatically receive patch updates, etc.

File: 1.0.3/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+flask==1.0.3
```
Comment (rbreslow) [inline 1.0.3/requirements.txt]: I would rename this folder to be `1.0`, but specify the patch release in `requirements.txt`.

**Review by rbreslow — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (fungjj92) [reply .travis.yml]: Oh cool! I will do that.

**Review by rbreslow — APPROVED** [review]
This looks good. 👍

I verified that the images being produced by Travis are tagged correctly, e.g. `1.0-python3.6-slim`.

While we're in here, it might be nice to do some housekeeping as well:

- Bump year in `LICENSE` to 2019
- Bump `gevent` and `gunicorn` versions for 0.11 and 0.12
- Bump example in `README.md`

**Comment by fungjj92** [comment]
I glanced through all the files and no other hairs seem out of place. Good call on those housekeeping details.

**Pull request merged**